static/js/upload.js

```diff
@@ -167,7 +170,7 @@ exports.options = function (config) {
         },
         raw_droppable: ['text/uri-list', 'text/plain'],
         drop: uploadStarted,
-        progressUpdated: progressUpdated,
+        globalProgressUpdated: globalProgressUpdated,
```

Comment:
I've pushed another commit, in which we now use `globalPrgressUpdated` to track the overall progress of the all the files, which solves the dangling progress bar when many files are uploaded at a time. I've seen some minor delays in completing and inserting of link in composebox so it would be better to do some manual testing before merging this commit as I'm not sure whether it's my system or it happens before as well.